lms/djangoapps/instructor/views/api.py

```diff
@@ -630,14 +630,8 @@ def students_update_enrollment(request, course_id):
     email_students = _get_boolean_param(request, 'email_students')
     is_white_label = CourseMode.is_white_label(course_id)
     reason = request.POST.get('reason')
-    if is_white_label:
-        if not reason:
-            return JsonResponse(
-                {
-                    'action': action,
-                    'results': [{'error': True}],
-                    'auto_enroll': auto_enroll,
-                }, status=400)
```

Comment:
I wonder why this check was here? Do we know how this gets called?

Reply:
Reason field was exposed only if is_white_label was True. Now we are showing this field globally without a check, but as you said, its not required on the backend API, hence we don't need this check and the corresponding test: test_reason_field_should_not_be_empty